prql-compiler/src/cli.rs

```diff
@@ -197,4 +197,37 @@ sort full
         sort full                                              # [?, full, gender]
         "###);
     }
+
+    #[test]
+    fn format_test() {
+        let output = Cli::execute(
```

Comment:
FYI there is an insta library for CLI now! (I added a link to the issue on CLI tests, can find it when back on a laptop)

Reply:
:+1: